packages/server/modules/activitystream/services/accessRequestActivity.ts

```diff
@@ -1,8 +1,6 @@
-import {
-  ActionTypes,
-  ResourceTypes,
-  saveActivity
-} from '@/modules/activitystream/services'
+import { saveActivity } from '@/modules/activitystream/services'
```

Comment:
I like to create special `addXActivity` functions for each activity type, because besides the resourceType & actionType you also need to ensure the info object is set up correctly etc. Easier to just invoke `addCommentCreatedActivity` than remember all of the parameters and values each time you invoke `saveActivity`, which is very low level.

Examples:
`modules/activitystream/services/streamActivity.ts`
`modules/activitystream/services/accessRequestActivity.ts`

Reply:
This is the file you just mentioned in the comment :'D

Reply:
> modules/activitystream/services/accessRequestActivity.ts

This is the same file you are referring to :D

Reply:
@gjedlicska oops, never mind, I thought you were adding a new activity item type. ignore this!